README.md

```diff
@@ -827,7 +827,9 @@ consider](https://trac.ietf.org/trac/iesg/wiki/ExpertTopics).
 The IESG has made a
 [statement](https://www.ietf.org/about/groups/iesg/statements/on-inclusive-language/)
 recommending authors review [NISTIR 8366](https://doi.org/10.6028/NIST.IR.8366)
-for guidelines on using inclusive terminology.
+for guidelines on using inclusive terminology. Similar guidance exists for other
+RFC Streams. Authors are encouraged to familiarize themselves with and apply the
```

Comment:
reference to similar guidance will be highly appreciated here, I think.

Reply:
They all point to the same NIST doc

Reply:
Then I don't see the use of this sentence -"Similar guidance exists for other
RFC Streams". and kind of redundant.  and if they all are same then s/similar/same.

Reply:
Well, they all gave their own guidance, but it all points to NISTIR 8366

Reply:
ok.